letsencrypt/tests/cli_test.py

```diff
@@ -10,8 +10,10 @@
 import mock
 
 from letsencrypt import account
+from letsencrypt import cli
```

Comment:
Is there a reason to do it that way?

There were a ridiculous number copies of `from letsencrypt import cli` in this file, and I was also unsure about the effect of mock decorators running *before* the imports that they were mocking.  But I guess that happens in other place without breaking.